### DISCUSSION TITLE
Welcome to Practice-Bidding-Scenarios Discussions!

### DISCUSSION THREAD

**Discussion by ADavidBailey** [ADavidBailey/Practice-Bidding-Scenarios#1] [Announcements]
## 👋 Welcome!
  I hope you will use Discussions to ask questions and make suggestions.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ADavidBailey** [comment]
BBO bidding tables are a fantastic resource to practice bidding, either with a partner or with a robot. But to make the most of the Bidding Tables one must know how to [program the dealer generator](https://www.bridgebase.com/tools/dealer/Manual/input.html) that BBO uses. I have created a new tool called Practice Bidding Scenarios that simplifies this process, providing dozens of canned bidding scenarios, for example. Jacoby Transfers, Smolen, slam bidding after a Stayman response, etc. Practice Bidding Scenarios piggybacks off the BBOalert browser extension to load the necessary dealer code, allowing many bidding scenarios to be selected with the press of a button.

![Bidding-Scenarios-1](https://github.com/ADavidBailey/Practice-Bidding-Scenarios/assets/146357155/c2c55534-921f-4c0f-8c7d-07972dc7e75f)

**Reply by JFrizzBridge** [reply]
Thinking best to use this to communicate?  This is straight from Gavin's lesson.

![image](https://github.com/ADavidBailey/Practice-Bidding-Scenarios/assets/149210059/b5449fd0-b8bc-4d26-9a10-33d7bc561f37)

![image](https://github.com/ADavidBailey/Practice-Bidding-Scenarios/assets/149210059/67b57de3-f6cb-4747-9231-f67c69d1b8bd)

**Reply by ADavidBailey** [reply]
Thanks, John.

So, what does it mean to Transfer to clubs or diamonds?  I assume you could transfer and pass. What if you transfer and bid again?

**Comment by ADavidBailey** [comment]
I watched Gavin's Transfers after 1M (X).  I'll do something with it.

On Fri, Oct 27, 2023 at 5:49 PM JFrizzBridge ***@***.***>
wrote:

> Thinking best to use this to communicate?
> [image: image]
> <https://user-images.githubusercontent.com/149210059/278774276-b5449fd0-b8bc-4d26-9a10-33d7bc561f37.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ADavidBailey/Practice-Bidding-Scenarios/discussions/1#discussioncomment-7408049>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BC4TXI3AYREHZIHZSE5ZWCDYBQ3BNAVCNFSM6AAAAAA5M3NFZ2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMBYGA2DS>
> .
> You are receiving this because you authored the thread.Message ID:
> <ADavidBailey/Practice-Bidding-Scenarios/repo-discussions/1/comments/7408049
> @github.com>
>


-- 
David Bailey
918-520-9700 mobile

**Reply by JFrizzBridge** [reply]
ok thx.  Pretty much current Expert Standard in the US, at least he says it is.

**Comment by ADavidBailey** [comment]
Here's the very first hand I got when testing w/3 robots.   What a hoot!


https://tinyurl.com/yq3n3yct

On Sat, Oct 28, 2023 at 10:24 AM JFrizzBridge ***@***.***>
wrote:

> ok thx. Pretty much current Expert Standard in the US, at least he says it
> is.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ADavidBailey/Practice-Bidding-Scenarios/discussions/1#discussioncomment-7411848>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BC4TXI2GPLPXPO2UXUOOYCLYBUPUFAVCNFSM6AAAAAA5M3NFZ2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMJRHA2DQ>
> .
> You are receiving this because you authored the thread.Message ID:
> <ADavidBailey/Practice-Bidding-Scenarios/repo-discussions/1/comments/7411848
> @github.com>
>


-- 
David Bailey
918-520-9700 mobile

**Comment by ADavidBailey** [comment]
I added it under both Major Suit Sequences and Competitive Auctions.  Try
it out and let me know what you think.

On Sat, Oct 28, 2023 at 11:06 AM David Bailey ***@***.***>
wrote:

> Here's the very first hand I got when testing w/3 robots.   What a hoot!
>
>
> https://tinyurl.com/yq3n3yct
>
> On Sat, Oct 28, 2023 at 10:24 AM JFrizzBridge ***@***.***>
> wrote:
>
>> ok thx. Pretty much current Expert Standard in the US, at least he says
>> it is.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/ADavidBailey/Practice-Bidding-Scenarios/discussions/1#discussioncomment-7411848>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BC4TXI2GPLPXPO2UXUOOYCLYBUPUFAVCNFSM6AAAAAA5M3NFZ2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMJRHA2DQ>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> <ADavidBailey/Practice-Bidding-Scenarios/repo-discussions/1/comments/7411848
>> @github.com>
>>
>
>
> --
> David Bailey
> 918-520-9700 mobile
>


-- 
David Bailey
918-520-9700 mobile

**Comment by JFrizzBridge** [comment]
import doesn't seem to load on Firefox on Windows 11

**Comment by ADavidBailey** [comment]
call me @ 918.520.9700

On Sat, Oct 28, 2023 at 12:35 PM JFrizzBridge ***@***.***>
wrote:

> import doesn't seem to load on Firefox on Windows 11
>
> On Fri, Oct 27, 2023 at 8:07 PM David Bailey ***@***.***>
> wrote:
>
> > Thanks, John.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/ADavidBailey/Practice-Bidding-Scenarios/discussions/1#discussioncomment-7408346>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/BDSMHS2DLVS7B2RHCP2DTTTYBREDJAVCNFSM6AAAAAA5M3NFZ2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMBYGM2DM>
>
> > .
> > You are receiving this because you commented.Message ID:
> >
> <ADavidBailey/Practice-Bidding-Scenarios/repo-discussions/1/comments/7408346
>
> > @github.com>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ADavidBailey/Practice-Bidding-Scenarios/discussions/1#discussioncomment-7412452>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BC4TXI6ZMAECXFJ42GEY7DDYBU647AVCNFSM6AAAAAA5M3NFZ2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMJSGQ2TE>
> .
> You are receiving this because you authored the thread.Message ID:
> <ADavidBailey/Practice-Bidding-Scenarios/repo-discussions/1/comments/7412452
> @github.com>
>


-- 
David Bailey
918-520-9700 mobile

**Comment by ADavidBailey** [comment]
I don't use Firefox (or Windows).  I know there are some tweaks that one
must make to use BBOalert with firefox.  See here:
https://github.com/stanmaz/BBOalert/blob/master/README.md#installation

On Sat, Oct 28, 2023 at 12:35 PM JFrizzBridge ***@***.***>
wrote:

> import doesn't seem to load on Firefox on Windows 11
>
> On Fri, Oct 27, 2023 at 8:07 PM David Bailey ***@***.***>
> wrote:
>
> > Thanks, John.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/ADavidBailey/Practice-Bidding-Scenarios/discussions/1#discussioncomment-7408346>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/BDSMHS2DLVS7B2RHCP2DTTTYBREDJAVCNFSM6AAAAAA5M3NFZ2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMBYGM2DM>
>
> > .
> > You are receiving this because you commented.Message ID:
> >
> <ADavidBailey/Practice-Bidding-Scenarios/repo-discussions/1/comments/7408346
>
> > @github.com>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ADavidBailey/Practice-Bidding-Scenarios/discussions/1#discussioncomment-7412452>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BC4TXI6ZMAECXFJ42GEY7DDYBU647AVCNFSM6AAAAAA5M3NFZ2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMJSGQ2TE>
> .
> You are receiving this because you authored the thread.Message ID:
> <ADavidBailey/Practice-Bidding-Scenarios/repo-discussions/1/comments/7412452
> @github.com>
>


-- 
David Bailey
918-520-9700 mobile

**Reply by JFrizzBridge** [reply]
Not a big deal, I don't really use it for BBO.  Usually Edge or Chrome

**Comment by ADavidBailey** [comment]
John, I don't recall what else you asked for.

And, I don't understand why you like this way of communicating.  Would you please explain.

**Reply by JFrizzBridge** [reply]
I prefer straight email for communications, keeping this thread clean.  message me your email via Facebook.

**Comment by ADavidBailey** [comment]
***@***.***

On Sat, Oct 28, 2023 at 10:29 PM JFrizzBridge ***@***.***>
wrote:

> I prefer straight email for communications, keeping this thread clean.
> message me you email via Facebook.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ADavidBailey/Practice-Bidding-Scenarios/discussions/1#discussioncomment-7414039>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BC4TXIZYLX6QTJK4HWRPQ7DYBXESZAVCNFSM6AAAAAA5M3NFZ2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMJUGAZTS>
> .
> You are receiving this because you authored the thread.Message ID:
> <ADavidBailey/Practice-Bidding-Scenarios/repo-discussions/1/comments/7414039
> @github.com>
>


-- 
David Bailey
918-520-9700 mobile
